teerex/src/lib.rs

```diff
@@ -364,10 +364,11 @@ pub mod pallet {
 						"DCAP quote verified. FMSPC from quote: {}",
 						hex::encode(fmspc)
 					);
-					match <SgxTcbInfo<T>>::get(fmspc) {
+					let sgx_status = match <SgxTcbInfo<T>>::get(fmspc) {
 						Some(reference) =>
-							if reference.verify_examinee(&tcb_info) {
+							if let Some(status) = reference.verify_examinee(&tcb_info) {
 								log::trace!("TCB info verification passed");
```

Comment:
how about adding the status into the trace log?

Reply:
see above